__docs__/wonder-blocks-banner/banner.stories.tsx

```diff
@@ -122,6 +122,9 @@ export const Kinds: StoryComponentType = {
 };
 
 /**
+ * DEPRECATED: The `layout` prop is deprecated and will be removed in a future
```

Comment:
The layout prop is still accepted so it isn't a breaking change, but it isn't being used. Created https://khanacademy.atlassian.net/browse/WB-1979 for removing the `layout` prop in our cleanup epic!